src/common/filesystem.c

```diff
@@ -1411,7 +1411,7 @@ void FS_BuildGenericSearchPath(void) {
 	fsRawPath_t *search = fs_rawPath;
 
 	while (search != NULL) {
-		Com_sprintf(path, sizeof(path), "%s/%s", search->path, BASEDIRNAME);
+		Com_sprintf(path, sizeof(path), "%s%s", search->path, BASEDIRNAME);
```

Comment:
the slash must stay: two slashes are harmless, but we could run into the case of having no slash at the end of the path if it has been specified by the user via cvar